src/components/AttachmentPicker/propTypes.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
```

Comment:
Can we rename this file to `AttachmentPickerPropTypes.js` to match the convention we have for other files of this type?